forloop_modules/function_handlers/database_handlers.py

```diff
@@ -505,9 +524,26 @@ def execute(self, node_detail_form):
         #     raise HTTPException(status_code=response.status_code, detail="Error requesting new node from api")
         
     def direct_execute(self, db_name, db_table_name, select, where_column_name, where_operator, where_value, limit, new_var_name):  
+        # Parse all combo-entry inputs to handle values from pipeline runs
+        db_name = parse_comboentry_input(db_name)
+        db_table_name = parse_comboentry_input(db_table_name)
+        select = parse_comboentry_input(select)
+        where_column_name = parse_comboentry_input(where_column_name)
+        where_operator = parse_comboentry_input(where_operator)
+
+        # Defensively strip whitespace/invisible characters from all string parameters
+        if isinstance(where_column_name, str): where_column_name = where_column_name.strip()
+        if isinstance(where_operator, str): where_operator = where_operator.strip()
+        if isinstance(where_value, str): where_value = where_value.strip()
+        if isinstance(limit, str): limit = limit.strip()
+        if isinstance(new_var_name, str): new_var_name = new_var_name.strip()
```

Comment:
input parsing and stripping shouldnt happen in direct_execute most probably, but I will forgive it

Reply:
This should rather be in "execute" and if it's general then in those "node_detail_form.get_chosen_value_by_name" etc. methods